### PR TITLE
Double scroll handled (fixes #309)

### LINKED ISSUE #309: Two scrollbars are seen on the work item list page
Two scrollbars are seen on the work item list page - http://demo.almighty.io/work-item-list
twoscrollbars

### PR DESCRIPTION
**Pull request by sanbornsen**

### CODE REVIEW

File: src/app/app.component.html [hunk 1]
```diff
@@ -1,3 +1,5 @@
+<div class="fix-container">
```
Comment (nimishamukherjee) [inline src/app/app.component.html]: The bottom of the scrollbar is not seen. If you try a resolution of 320x480, the scrollbar is not seen after add work item. 




Even in higher resolution, the bottom scroll arrow is not seen. Also the last work item is partially hidden.

Comment (sanbornsen) [inline src/app/app.component.html]: Yeah, so this would be another issue to fix the body.

Comment (sanbornsen) [inline src/app/app.component.html]: @SMahil you can take this up, create a new issue and work on it.
Also, we are not going to focus on mobile.

**Comment by joshuawilson** [comment]
@SMahil please review and approve if you agree.

**Pull request closed without merging**